src/pybind/mgr/cephadm/module.py

```diff
@@ -87,8 +86,6 @@ def remoto_has_connection(self):
   ConnectTimeout=30
 """
 
-CEPH_DATEFMT = '%Y-%m-%dT%H:%M:%S.%fZ'
```

Comment:
Note that `CEPH_DATEFMT` is the format used in the monmap, not the format which is used for json generation.